portfolio/src/main/webapp/style.css

```diff
@@ -38,11 +45,4 @@
 .button:hover {
   background-color: darkgray;
   color: white;
-}
-
-.comment-form {
-  padding: 20px;
-  width: 270px;
-  background-color: white;
-  border-color: darkgray;
 }
```

Comment:
It's complaining about lack of a newline at the end of the file. I'd check here and elsewhere.

Reply:
I'm not seeing a newline issue?

Reply:
https://screenshot.googleplex.com/2RAboSJJ9Z0